runtime/array.inl

```diff
@@ -1488,8 +1494,17 @@ void array<T>::unset(const string &string_key) {
     return;
   }
```

Comment:
This if() can be deleted?

Reply:
Sure, but in this case we get small performance issue - hash will be calculated even for array that is actually vector. Given that is_vector() much cheaper then string_key.hash() I think better leave as is.

Reply:
Yes, you're right, but partially. Because unsetting from a vector by a string key is a very strange operation, that will immediately lead to converting this vector-array to a hashmap-array :)
But ok, let's leave it